receiver/awscontainerinsightreceiver/internal/neuron/neuron_empty_metric_decorator.go

```diff
@@ -159,7 +166,38 @@ func createNewMetricFromHardwareInfo(hardwareInfo pmetric.Metric, metricName str
 	metricBody := metricToAdd.Gauge().DataPoints().At(0)
 	metricBody.SetDoubleValue(0)
 	metricBody.Attributes().PutStr("runtime_tag", RuntimeTagOverride)
-	metricBody.Attributes().PutStr("runtime_tag", RuntimeTagOverride)
 
 	return metricToAdd
 }
+
+// method to add neuroncore_per_device_count attribute to all metrics
+func (ed *EmptyMetricDecorator) addNeuronCorePerDeviceAttribute(metrics pmetric.MetricSlice, neuronCoresPerDevice int) {
```

Comment:
Why does this have to be a datapoint attribute? Cant it be a resource attr?

Reply:
Its the same pmetric getting passed to the processor right? I dont get why it wont be available later on.
Please track this as a to-do.